test-suite/LocalDebugger/LocalDebugger.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>

```

Comment:
You need to revert all such changes, it depends on runtime that is going to be used to run tests and we use different runtimes. To handle this we simply use `sed -i "s/netcoreapp${hardcoded_test_clr_version}/netcoreapp${test_clr_version}/g"` before launch of tests.

Reply:
Oh, I never saw this, let me do it straightaway.